io.github.dagargo.Elektroid.yaml

```diff
@@ -0,0 +1,42 @@
+id: io.github.dagargo.Elektroid
+runtime: org.gnome.Platform
+runtime-version: '43'
+sdk: org.gnome.Sdk
+command: elektroid.sh
+finish-args:
+  - "--socket=fallback-x11"
+  - "--socket=wayland"
+  - "--socket=pulseaudio"
+  - "--filesystem=home"
+  - "--device=all"
+rename-desktop-file: elektroid.desktop
+rename-appdata-file: elektroid.appdata.xml
+rename-icon: elektroid
+copy-icon: true
+separate-locales: false
+modules:
+  - name: libzip
+    buildsystem: cmake-ninja
+    sources:
```

Comment:
as a general style, `sources` is the last item in the `modules`.

Reply:
Changed in bc3a8bd671707691e5583460611f6beb58a3824a.